plugins/modules/aws_secret.py

```diff
@@ -54,6 +53,13 @@
     - Specifies string or binary data that you want to encrypt and store in the new version of the secret.
     default: ""
     type: str
+  resource_policy:
+    description:
+    - Specifies JSON-formatted resource policy to attach to the secret. Useful when granting cross-account access
+       to secrets.
+    required: false
+    type: json
+    version_added: 2.2.0
```

Comment:
@markuman would this be `2.3.0` now ?

Reply:
if version_added means collection version, the next one is 3.0.1. @markuman, could you please advise when you get a chance? Thanks.

Reply:
I guess 3.0.1 has been released. I would probably specify 3.1.0 if that makes sense for @markuman.

Reply:
Oh yes. The release train is moving fast this year.  
+1 for 3.1.0

Reply:
Thank you, @markuman and @alinabuzachis . Bumped the version_added to 3.1.0